nrf52811-hal/Cargo.toml

```diff
@@ -26,10 +26,7 @@ default-features = false
 features = ["52811"]
 version = "=0.16.1"
 
-[dependencies.embedded-hal]
-features = ["unproven"]
-version = "0.2.3"
-
 [features]
+embedded-hal-02 = ["nrf-hal-common/embedded-hal-02"]
 rt = ["nrf52811-pac/rt"]
-default = ["rt"]
+default = ["rt", "embedded-hal-02"]
```

Comment:
I guess `embedded-hal-02` is added as a default feature to avoid breaking change. But what about users that disable default features? I'm not sure what the [Cargo Book](https://doc.rust-lang.org/cargo/reference/semver.html) recommends in those cases.

Reply:
There will be breaking changes in this release anyway, so I'm not really worried about that. I've included it by default just to make life easier for new users who might be using device drivers that depend on the embedded-hal 0.2 traits. People who care about binary size and build time can disable default features.